kolibri/plugins/facility/assets/src/views/FacilityConfigPage/index.vue

```diff
@@ -146,13 +166,16 @@
       FacilityAppBarPage,
       ConfirmResetModal,
       EditFacilityNameModal,
+      BottomAppBar,
+      CreateManagementPinModal,
     },
     mixins: [commonCoreStrings, responsiveWindowMixin],
     data() {
       return {
         showModal: false,
         showEditFacilityModal: false,
         settingsCopy: {},
+        CreatePinShow: false,
```

Comment:
Very small nit-pick, I would change this to `createPinShow`, just for consistent cases